app/views/layouts/application.html.erb

```diff
@@ -12,13 +12,25 @@
     <div class="navbar">
       <%= render "layouts/navbar" %>
     </div>
-    <div class="body">
-      <% flash.each do |message_type, message| %>
-        <div class="alert alert-<%= message_type %>">
-          <%= message %>
+    <div class="container-fluid">
+      <% if logged_in? %>
+        <div id="wrapper">
+          <%= render "layouts/sidebar" %>
+          <div class="page-content-wrapper">
+            <% flash.each do |message_type, message| %>
+              <div class="alert alert-<%= message_type %> col-4"><%= message %></div>
+            <% end %>
+            <%= yield %>
+          </div>
+        </div>
+      <% else %>
+        <div class="container page-content-wrapper">
+          <% flash.each do |message_type, message| %>
+            <div class="alert alert-<%= message_type %> text-center"><%= message %></div>
+          <% end %>
+          <%= yield %>
```

Comment:
Code này lặp quá chú, list code này đưa khỏi if - else đi hì 

khác nahu về css em thì dùng toán tử 3 ngôi đi hì